### ISSUE TITLE
logical processor metric output is not matching  with server

### ISSUE THREAD

**Issue by krdineshbny** [prometheus-community/windows_exporter#1593]
### Current Behavior

windows exporter
windows_cs_logical_processors is showing count as 24 but actual in server is 152

### Expected Behavior

logical processors is 152

### Steps To Reproduce

```Markdown
Get-WmiObject -class Win32_processor | ft NumberOfCores,NumberOfLogicalProcessors
```

### Environment

- windows_exporter Version: 0.25.1
- Windows Server Version: windows server 2019

### windows_exporter logs

```shell
count mismatch
```

### Anything else?
![processor details](https://github.com/user-attachments/assets/8e35aedd-7775-47e2-bbd9-5101879cf37d)
![processor details_powershell](https://github.com/user-attachments/assets/ae9b4cc3-0cf3-4c4b-bfa8-b3c73d1eb561)
![processor details_using_cpu_metric](https://github.com/user-attachments/assets/812602ab-4395-4b53-a615-2e2f696c0c0b)

![processor details_cs_logical_metric](https://github.com/user-attachments/assets/c8a7cc8f-c75e-45b2-84b4-2210cbfe6e20)

**Comment by jkroepke** [comment]
Could it be possible that the server has NUMA based Processor Groups?

Currently, windows_exporter uses https://learn.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getsysteminfo and the function may only return the number of processors from the current group.

![image](https://github.com/user-attachments/assets/dcdb0a03-c425-42ef-8012-efad00e4a911)

**Comment by jkroepke** [comment]
Fixed in #1596
